pkg/storage/utils/decomposedfs/node/permissions_darwin.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018-2021 CERN
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// In applying this license, CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+//go:build darwin
+// +build darwin
+
+package node
+
+import (
+	"syscall"
+
+	"github.com/pkg/xattr"
+)
+
+func isNoData(err error) bool {
```

Comment:
Tbh, I always find code flow easier to understand if funcs like this are non negating, rather `isEmpty()` than `isNotEmpty()`.
In this case it would mean to turn it around to `isData()` which is also kind of non speaking name. 

As I understand what it does, it should be something like `isNoattrErrno()`. Maybe.

Reply:
done